src/Stratis.Bitcoin.Features.BlockStore/BlockStoreSignaled.cs

```diff
@@ -13,10 +16,7 @@ namespace Stratis.Bitcoin.Features.BlockStore
 {
     public class BlockStoreSignaled : SignalObserver<Block>
     {
-        private readonly IAsyncLoopFactory asyncLoopFactory;
-
-        /// <summary>The async loop we need to wait upon before we can shut down this feature.</summary>
-        private IAsyncLoop asyncLoop;
+        private Task relayWorkerTask;
```

Comment:
pls comment all these new guys

also could you put the constant above fields?

Reply:
Fixed